.devcontainer/Dockerfile

```diff
@@ -0,0 +1,42 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+ARG IMAGE_TYPE=latest-cpu
```

Comment:
This file should be removed.

Reply:
https://github.com/keras-team/keras/blob/master/.devcontainer/Dockerfile This file doesn't have a MS Copyright.

Reply:
As it was contributed by me in the Microsoft/Vscode repos let's check what is the @chuxel position on this for the CLA

Reply:
P.s. the original PR was submitted in this repo before the vscode/MS one after the rejection:
https://github.com/tensorflow/addons/pull/1309/files

Reply:
This is your Dockefile, so no need to include the copyright.

Reply:
Ok I've remote the MS copyright here.